platform/nephos/python-saithrift.mk

```diff
@@ -0,0 +1,6 @@
+# python-saithrift package
+
+PYTHON_SAITHRIFT_NEPHOS = python-saithrift_0.9.4_amd64.deb
```

Comment:
why still 0.9.4?

Reply:
@lguohan , we followed [Broadcom platform](https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/python-saithrift.mk) and saw that each platform vendor using 0.9.4 in this file as well.

Reply:
ok. let's keep as it is for now. We'll revisit this issue again to upgrade the saithrift to 1.2.